README.md

```diff
@@ -77,6 +79,18 @@ Start the service
 npm run start
 ```
 
+### Using an image cache service
+
+The Token Metadata Service allows you to specify the path to a custom script that can pre-process
+every image URL detected by the service before it's inserted into the DB. This allows you to serve
+CDN image URLs in your metadata responses instead of raw URLs which may become unreachable with
+time.
```

Comment:
Is it worth mentioning that this feature should be enabled for user-privacy reasons? E.g. for similar reason to why gmail proxies external images in emails to thwart tracking pixels.

Reply:
Good idea, done